core/src/main/java/org/apache/spark/raydp/RayDPUtils.java

```diff
@@ -34,4 +37,16 @@ public static <T> ObjectRefImpl<T> convert(ObjectRef<T> obj) {
       throw new RuntimeException(obj.getClass() + " is not ObjectRefImpl");
     }
   }
+
+  /**
+   * Create ObjectRef from Array[Byte] and register ownership.
+   * We can't import the ObjectRefImpl in scala code, so we do the conversion at here.
+   */
+  public static <T> ObjectRef<T> readBinary(byte[] obj, Class<T> clazz, byte[] ownerAddress) {
+    ObjectId id = new ObjectId(obj);
+    ObjectRefImpl<T> ref = new ObjectRefImpl<>(id, clazz);
+    ((RayRuntimeInternal) Ray.internal()).getObjectStore()
+        .registerOwnershipInfoAndResolveFuture(id, null, ownerAddress);
```

Comment:
`registerOwnershipInfoAndResolveFuture` is invoked in the constructor of ObjectRefImpl. I don't think we should add it any more.